app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
```

Comment:
You have not yet fixed them

Reply:
i have fixed it again
![image](https://user-images.githubusercontent.com/68294917/89989834-13a49a80-dcb4-11ea-954a-1031271741dd.png)